tslint.json

```diff
@@ -71,7 +71,6 @@
     "no-switch-case-fall-through": true,
     "no-trailing-whitespace": true,
     "no-unnecessary-initializer": true,
-    "no-use-before-declare": true,
```

Comment:
This rule seems to make sense, why it has been removed?

Reply:
It got removed together with a Typescript version bump as it is deprecated since `2.9` - see https://github.com/palantir/tslint/pull/4695. The explanation for the removal is [here](https://palantir.github.io/tslint/rules/no-use-before-declare/) (tl;dr - it isn't useful when you don't use `var` keyword, and we don't in the integration code). However, as mentioned in [this comment](https://github.com/palantir/tslint/pull/4695#issuecomment-489455065), there are still cases where it may be useful, so I'll bring it back.

Reply:
Didn't know that, nice research :+1: But if it has been deprecated, maybe we should keep it removed then?

Reply:
Well, until we fix #30 it doesn't really matter 😄  but sure, let's keep it away while we can and if we encounter a valid use case for it in our code we can restore it.